### PR TITLE
Content fill

### PR DESCRIPTION
**Pull request by Starvosky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,50 +22,96 @@
...
-      <!-- Hobbies -->
-    </article>
+        <h4>Conestoga Valley High School</h4>
+        <h5>Lancaster, Pennsylvannia</h5>
+        <ul>
+          <li>Graduated June 2015</li>
+        </ul>
 
-    <footer>
+        <h4>Lancaster County Career & Technology Center</h4>
```
Comment (calebgmahala) [inline index.html]: you need to use &amp;

File: index.html [hunk 2]
```diff
@@ -20,50 +22,96 @@
...
+        <h4>Thaddeus Stevens College of Technology</h4>
+        <h5>Lancaster, Pennsylvannia</h5>
+        <ul>
+          <li>Expected to graduate June 2019 from CSET with an associate's degree.</li>
+        </ul>
+      </div>
+
+      <div>
+        <h4>Skills</h4>
+        <p>
```
Comment (calebgmahala) [inline index.html]: don't need a p tag

Comment (calebgmahala) [reply index.html]: you already have list below it.

**Review by edgararodriguez — APPROVED** [review]
Everything looks good just get rid of the P tag and the content looks fine

**Pull request merged**